app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/dashboard/DashboardScreen.kt

```diff
@@ -41,7 +59,74 @@ fun DashboardScreen(
                 }
             },
             content = { padding ->
-                Box(modifier = Modifier.padding(padding)) {}
+                Box(modifier = Modifier.padding(padding)) {
+                    Column{
+                        Card(modifier = Modifier.padding(padding)) {
+                            Card(
+                                modifier = Modifier.fillMaxWidth(),
+                                shape = Shapes.small,
+                                colors = CardDefaults.cardColors(containerColor = DashBoardBackgroundPurple)
+                            ) {
+                                Header(
+                                    text = "Upcoming",
+                                    textColor = DashBoardHeader,
+                                    textStyle = MaterialTheme.typography.titleLarge,
+                                    modifier = Modifier.padding(16.dp),
+                                    fontWeight = FontWeight.Bold
+                                )
+                                LazyRow(
+                                    modifier = Modifier.padding(start = 16.dp, bottom = 16.dp),
+                                    horizontalArrangement = Arrangement.spacedBy(16.dp),
+                                    verticalAlignment = Alignment.CenterVertically
+                                ) {
+                                    items(5) { UpcomingInterviewCard() }
```

Comment:
I would suggest move all mock data in future PRs to a separate file, so that it would be easy to delete all mock data when we will have actual implementation.